src/pyscipopt/recipes/primal_dual_evolution.py

```diff
@@ -0,0 +1,50 @@
+from pyscipopt import Model, Eventhdlr, SCIP_EVENTTYPE, Eventhdlr
+
+def get_primal_dual_evolution(model: Model):
```

Comment:
maybe the name should be `attach_primal_dual_evolution_eventhdlr`? longer but more descriptive of what it is